### PR TITLE
Make it possible to use different az for instances creating via ecs_node role

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Make it possible to use different az for instances creating via ecs_node role

### CODE REVIEW

File: roles/ecs_node/vars/main.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+az_length: "{{ ecs_availability_zones | length }}"
```
Comment (outcatcher) [inline roles/ecs_node/vars/main.yaml]: It would be better to call it `az_count` :)

File: roles/ecs_node/tasks/build.yaml [hunk 2]
```diff
@@ -60,3 +60,4 @@
     delete_fip: true
     auto_ip: false
   with_sequence: start=1 end="{{ ecs_count|int }}"
+
```
Comment (outcatcher) [inline roles/ecs_node/tasks/build.yaml]: Is this empty line needed?

File: roles/ecs_node/defaults/main.yaml [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 ecs_volume_size: 5
 ecs_image: ""
-ecs_count: ""
 ecs_network_id: ""
 ecs_flavor: ""
 ecs_key_pair_name: ""
 ecs_scenario_name: "ecs_node"
-ecs_secgroup_ports: "[ 22, 80, 443 ]"
+ecs_secgroup_ports: [ "22", "80", "443" ]
+ecs_count: ""
```
Comment (outcatcher) [inline roles/ecs_node/defaults/main.yaml]: Maybe set this value to `1` by default?

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: roles/ecs_node/vars/main.yaml (hunk 1 above)
Comment (Polina-Gubina) [reply roles/ecs_node/vars/main.yaml]: done

File: roles/ecs_node/tasks/build.yaml (hunk 2 above)
Comment (Polina-Gubina) [reply roles/ecs_node/tasks/build.yaml]: done

File: roles/ecs_node/defaults/main.yaml (hunk 3 above)
Comment (Polina-Gubina) [reply roles/ecs_node/defaults/main.yaml]: done

**Review by anton-sidelnikov — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**